src/hughes/todo/ToDoCalendar.js

```diff
@@ -1,15 +1,48 @@
 foam.CLASS({
   package: 'hughes.todo',
   name: 'ToDoCalendar',
-
-  properties: [
+	
+	implements: [
```

Comment:
need to setup vim to replace tab with spaces so we only get two spaces here and not two tabs